src/main/scala/nl/knaw/dans/easy/multideposit/CommandLineOptions.scala

```diff
@@ -66,7 +66,7 @@ object CommandLineOptions {
   }
 }
 
-class ScallopCommandLine(props: => PropertiesConfiguration, args: Array[String]) extends ScallopConf(args) {
+class ScallopCommandLine(props: PropertiesConfiguration, args: Array[String]) extends ScallopConf(args) {
```

Comment:
Given earlier changes, we don't need the call-by-name part of this parameter anymore